CMakeLists.txt

```diff
@@ -224,6 +224,12 @@ if (JEMALLOC_FOUND)
   set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -ljemalloc -L${JEMALLOC_LIBRARY_PATH}")
 endif()
 
+#---Here we add liburing to the linker flags if needed------------------------------------------
+if (LIBURING_FOUND)
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -luring -L${LIBURING_LIBRARY_PATH}")
```

Comment:
Can we not add this to the library flags of `RIO` as the one containing `RIoUring`, instead of putting it everywhere, whether the binary links against `RIO` or not?

Reply:
Thank you, I took this out in 015977dfea